### PR TITLE
Bug 1967933: Download must-gather pod logs to get stdout info

### PR DESCRIPTION
**Pull request by tssurya**
Network-tools image leverages the must-gather tool
using the "--image" option. However some of the scripts
and tools used there just print the output to stdout.
It would be nice to get the must-gather pod logs as
well downloaded locally to see what happened while
the data was gathered or scripts were being run.

### CODE REVIEW

**Review by soltysh — CHANGES_REQUESTED** [review]
/hodl

File: pkg/cli/admin/mustgather/mustgather.go [hunk 1]
```diff
@@ -380,6 +381,12 @@ func (o *MustGatherOptions) Run() error {
 				errCh <- fmt.Errorf("unable to download output from pod %s: %s", pod.Name, err)
 				return
 			}
+			// copy the must-gather pod logs into the local destination dir
```
[the remark this answers is not in the document]
Comment (tssurya) [reply pkg/cli/admin/mustgather/mustgather.go:384]: Thanks so much for having a look on short notice!

hmm ok I was told by Eric that there was plans to get this done. Must be some kind of miscommunication then! 

Aim was for both, to investigate if something fails and also for the stdout to get collected.

[the remark this answers is not in the document]
Comment (soltysh) [reply pkg/cli/admin/mustgather/mustgather.go:384]: must-gather container collection is possible, but shouldn't be bet on, I'd start with always gathering data into files.

[the remark this answers is not in the document]
Comment (tssurya) [reply pkg/cli/admin/mustgather/mustgather.go:384]: right I agree, but there is this slight other problem which is we are planning to use that image to run the scripts as a daemonset as well. So the image needs to be compatible with both must-gather and running as daemonset which is why this patch would be useful for us.

[the remark this answers is not in the document]
Comment (soltysh) [reply pkg/cli/admin/mustgather/mustgather.go:384]: I'd imagine a flag which would properly work for either of the two on the script side.

**Pull request closed without merging**